docs/ref/settings.txt

```diff
@@ -836,6 +836,23 @@ This is an Oracle-specific setting.
 
 The maximum size that the DATAFILE_TMP is allowed to grow to.
 
+.. setting:: DATA_UPLOAD_MAX_MEMORY_SIZE
+
+DATA_UPLOAD_MAX_MEMORY_SIZE
+---------------------------
+
+.. versionadded:: 1.9
+
+Default: ``2621440`` (i.e. 2.5 MB).
+
+The maximum size (in bytes) that a request body may be before raising a
+:exc:`~django.core.exceptions.SuspiciousOperation` error. This value is applied
+when accessing ``request.body`` or ``request.POST`` and is calculated against
+the total request size excluding any file upload data. You can set it to
+``None`` to disable the check.
```

Comment:
I think it would be helpful to give some guidance on cases where the default value may need to be increased. (large forms?)